dev/doxygen/pages/translations.md

```diff
@@ -0,0 +1,22 @@
+Translation Guidelines {#doc_translations}
+========================================
```

Comment:
line too short :)

Reply:
This is exactly why underlining sucks in Markdown :) The hash prefix variant is much nicer in that regard.